src/test/java/org/junitpioneer/jupiter/resource/PathDeleterTests.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2016-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.resource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.io.IOException;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import com.google.common.jimfs.Configuration;
+import com.google.common.jimfs.Jimfs;
+
+import org.junit.jupiter.api.Test;
+
+class PathDeleterTests {
+
+	@Test
+	void deletesFile() throws IOException {
+		try (FileSystem fileSystem = Jimfs.newFileSystem(Configuration.unix())) {
+			Path file = fileSystem.getPath("file.txt");
+			Files.createFile(file);
+
+			Files.walkFileTree(file, PathDeleter.INSTANCE);
+
+			assertThat(file).doesNotExist();
+		}
+	}
```

Comment:
Added tests to verify things actually get deleted. 😉

Reply:
Thanks!

These tests are not strictly needed because the deletion functionality in `PathDeleter` is already covered by `TemporaryDirectory`'s tests. However these new tests act as a nice sanity check in case the `TemporaryDirectory` tests are ever changed or removed, so I'm happy to keep these. :)